.github/workflows/playwright_ui_tests.yml

```
@@ -21,7 +21,7 @@ jobs:
       - name: Setup Node.js
         uses: actions/setup-node@v4
         with:
-          node-version: "20"
+          node-version: 20
```

```suggestion
          node-version: lts/*
```
In Visualizer CI we use lts version